Dockerfile

```diff
@@ -0,0 +1,14 @@
+### 1. Get python 3.6
+FROM python:3.6-alpine
+
+WORKDIR /tmp
+COPY ./requirements.txt /tmp
+
+### 2. Get required packages via the package manager
+RUN apk update \
```

Comment:
Why do we need openjdk8-jre? Is this a requirement of PySpark? If so, that well and truly scuppers the use of distroless :(

Reply:
Yes Matt  pyspark  needs java 8

Reply:
That's...disappointing :) But not entirely unexpected!